### PR TITLE
fix(starr): New French guide

### PR DESCRIPTION
**Pull request by SeiyaGame**
# Pull Request

## Purpose

PR to fix error found in the new french guide

Edit: PR is ready to merge, need review of @NiceTSY 

## Requirements

- [X] These changes meet the standards for [contributing](https://github.com/TRaSH-Guides/Guides/blob/master/CONTRIBUTING.md).
- [X] I have read the [code of conduct](https://github.com/TRaSH-Guides/Guides/blob/master/.github/CODE_OF_CONDUCT.md).

## TODO / Done

- [X] [(fix) Missing french-multi-vf score for web-scene](https://github.com/TRaSH-Guides/Guides/commit/799e37303e71f5459533bef51855a835c622ba0e)
- [X] [(fix) Missing tag \<sub\> for FR LQ in Sonarr Collection CF](https://github.com/TRaSH-Guides/Guides/commit/aab446a47b2ddd707d15f453909247145404d004)
- [X] Add Language Profile to Sonarr/Radarr Collection CF
- [ ] Add table for Language Profile in Sonarr Collection CF (Not for the moment)

### Not related to French Guide :

- [X]  Wrong link to Sonarr-Quality-Settings-File-Size in Anime Guide

### CODE REVIEW

**Comment by SeiyaGame** [comment]
@NiceTSY not so much fix, more on the anime part, but in progres in this PR #2133

**Review by SeiyaGame — COMMENTED** [review]
@TRaSH- Tell me if we can at least add the CF like this? So that people have access to the json

**Comment by SeiyaGame** [comment]
> > @TRaSH- Tell me if we can at least add the CF like this? So that people have access to the json
> 
> Doesn't the Radarr one need fixes also?

I completely forgot the part about Radarr but it's fixed, I tested it locally. (`mkdocs serve`)

**Comment by SeiyaGame** [comment]
@NiceTSY Does the Radarr `web-scene.json` need also to be fixed or it's not used in the Radarr FR Guide ?
 
This line is missing :
image

**Comment by NiceTSY** [comment]
> @NiceTSY Does the Radarr `web-scene.json` need also to be fixed or it's not used in the Radarr FR Guide ?
> 
> This line is missing : image

We can definitely add it to the Radarr FR Guide. Could you also remove the two `!` also from both Sonarr and Radarr for this CF?

**Comment by SeiyaGame** [comment]
I removed the `!` for sonarr but I didn't add a web-scene for Radarr because it doesn't work the same way.
Apparently the scene on radarr isn't even used in fallback

@NiceTSY For me it's all good
The rest of the fixes are for anime, which can be found here #2133

File: docs/Radarr/Radarr-collection-of-custom-formats.md
Comment (NiceTSY) [inline docs/Radarr/Radarr-collection-of-custom-formats.md:1]: @TRaSH- 
Just checking with you if you agree on putting them into the collection of CFs (but not into the tables of the same page). If so this PR is good for me to go.

File: includes/french-guide/sonarr-cf-french-web-scoring-en.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-    |---------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|:-------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------:|
+    | ------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- | :-----------------------------------------------------------------------------: | :-------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------: |
```
Comment (bakerboy448) [inline includes/french-guide/sonarr-cf-french-web-scoring-en.md:4]: Why are we reformatting this table?

File: includes/french-guide/sonarr-cf-french-web-scoring-fr.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
-    |---------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------|:-------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------:|
+    | ------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- | :-----------------------------------------------------------------------------: | :-------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------: |
```
Comment (bakerboy448) [inline includes/french-guide/sonarr-cf-french-web-scoring-fr.md:4]: Ditto

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

File: includes/french-guide/sonarr-cf-french-web-scoring-en.md (hunk 1 above)
Comment (SeiyaGame) [reply includes/french-guide/sonarr-cf-french-web-scoring-en.md:4]: My VS Code automatically adapts the table, but I don't see any particular problem? It's even better this way, isn't it?
I can revert this change if you want

Comment (SeiyaGame) [reply includes/french-guide/sonarr-cf-french-web-scoring-en.md:4]: it's also how I learned to make tables in markdown

**Review by TRaSH- — APPROVED** [review]
I tested locally, and the only error I found was something in the French anime section that would be fixed after the merge of PR #2133

Keep in mind the French language CF JSON is temp-approved and can be changed later.

**Comment by SeiyaGame** [comment]
> Keep in mind the French language CF JSON is temp-approved and can be changed later.

Yup no problem for that ! it was seen with @NiceTSY

**Pull request merged**